demo/src/features/imgui/internal/imgui_font_atlas.rs

```diff
@@ -23,7 +23,6 @@ impl ImguiFontAtlasData {
 #[derive(Clone)]
 pub struct ImguiFontAtlas(pub ResourceArc<ImageViewResource>);
 
-#[cfg(feature = "use-imgui")]
```

Comment:
Should this feature flag have been removed?

Reply:
this flag existed because sdl2 + imgui panics on iOS due to it trying to use mouse cursor APIs. Now that the logic for controlling the mouse cursor is in our own crate, we can just make it not do that on iOS. (I'll address it next time I make an iOS build)